src/toolbar_editor.cpp

```diff
@@ -28,10 +28,13 @@ Q_DECLARE_METATYPE(QMenu*)
 Q_DECLARE_METATYPE(QToolBar*)
 Q_DECLARE_METATYPE(QAction*)
 
+const QString Toolbar_Editor::customToolbarNamePrefix = QString("custom_toolbar_");
```

Comment:
Yes in my comment I explain a way to allow the caller to specify protected toolbars...